.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: release
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Verify branch name
+        if: github.ref != 'refs/heads/main'
+        run: |
+          echo "🚨 The release must start from the main branch!"
+          exit 1
+
+      - name: Configure releaser details
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Trigger release script
+        run: pnpm run release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I think this job needs to be a little bit more complicated, because we need to release on `npm` as well, right?

https://github.com/release-it/release-it/blob/main/docs/ci.md#github-actions

Reply:
You’re right, this would create a GitHub Release only 😅 I’ll update the code as per the docs you linked.